src/tests/localvocal-offline-test.cpp

```diff
@@ -43,9 +44,14 @@ void obs_log(int log_level, const char *format, ...)
 	std::time_t now_time_t = std::chrono::system_clock::to_time_t(now);
 	std::tm now_tm = *std::localtime(&now_time_t);
```

Comment:
think we don't need this anymore right?

Reply:
I kind of like having both timestamps available, local time to orient myself on when a particular run happened and "running time" to compare relative timing within a run